### PR TITLE
Decoupling FilmDrop k8s Terraform Repo

### PR DESCRIPTION
**Pull request by hectormachin**

### CODE REVIEW

File: inputs.tf [hunk 1]
```diff
@@ -1,80 +1,132 @@
...
+  default     = false
+  description = "Deploy Argo Events"
+}
+
+variable namespace_annotations {
+  type        = map(string)
+  description = "MAP of custom defined namespace annotations"
+}
+
+variable "deploy_titiler" {
```
Comment (geomatician) [inline inputs.tf:122]: maybe just change this to `variable deploy_titiler` to be consistent with the other variable names? (i.e. remove the quotes)

Comment (hectormachin) [reply inputs.tf]: Fixed, thank you!

**Comment by geomatician** [comment]
do we need the 'eks' folder if there is nothing in there anymore?

**Comment by hectormachin** [comment]
@geomatician > do we need the 'eks' folder if there is nothing in there anymore?

There is no EKS folder, take a look at the branch https://github.com/Element84/filmdrop-k8s-tf-modules/tree/hm/environment-restructure

**Review by paulpilone — APPROVED** [review]
Only suggestion would be to add configuring/adding a new profile to the operations guide. Otherwise looks good!

**Review by geomatician — APPROVED** [review]

**Pull request merged**